### PR TITLE
Makes Clippy happy

### PR DESCRIPTION
**Pull request by Tpt**

### CODE REVIEW

**Comment by Tpt** [comment]
The "[Uses inline variable names in fmt strings](https://github.com/paritytech/parity-db/commit/e9777bce0c49683a8e51af0456f1aa66e9df337c)" commit has been done automatically with `cargo clippy --fix`

File: src/column.rs [hunk 1]
```diff
@@ -409,10 +409,10 @@ impl HashColumn {
 		self.write_reindex_plan_locked(tables, reindex, key, address, log)
 	}
 
-	fn write_reindex_plan_locked<'a, 'b>(
+	fn write_reindex_plan_locked(
 		&self,
-		mut tables: RwLockUpgradableReadGuard<'a, Tables>,
-		mut reindex: RwLockUpgradableReadGuard<'b, Reindex>,
+		mut tables: RwLockUpgradableReadGuard<'_, Tables>,
+		mut reindex: RwLockUpgradableReadGuard<'_, Reindex>,
```
Comment (cheme) [inline src/column.rs:415]: Can they be just ignored? (or somehow having two _ indicate they are different)

**Review by cheme — APPROVED** [review]

File: src/column.rs (hunk 1 above)
Comment (Tpt) [reply src/column.rs]: Yes, they can. I have pushed an extra commit in this PR removing them (and some others)

Comment (cheme) [reply src/column.rs]: Thanks :)

**Pull request merged**